javascript/javascript1/week4/voiceAssistent.js

```diff
@@ -0,0 +1,229 @@
+let keyWordName = "name";
```

Comment:
there is a lot of good code in this file, but you missed the first part of the homework question
```
Create a function called getReply(command). The function should return a response that corresponds to the command!
```

The idea is that the getReply function is called like this, where --> is an output from console.log

```
console.log(getReply("Hello! my name is Russell"))
--> Nice to meet you Russell
console.log(getReply("What is my name"))
--> Your name is Russell
```

The function should probably be written something like this

```
let name = ""
function getReply(command) {
   if (command.startswith("Hello! my name is")) {
       name = getNameFromArray(getArrayFromString(command))
       return name
   } else if (command === "What is my name") {
       return getMyName(command, name)
   }
   // ....
}
```

Obviously there are heaps of commands missing, but the idea is to build a function that can handle all the commands